overrides/EmbedLiteOverrides.manifest

```diff
@@ -13,3 +13,4 @@ locale branding ru ru/locale/branding/
 locale browser ru ru/locale/ru/browser/
 override chrome://global/locale/netError.dtd chrome://browser/locale/netError.dtd
 override chrome://global/locale/appstrings.properties chrome://browser/locale/appstrings.properties
+
```

Comment:
```suggestion
```
For diff cleanliness, this change could be skipped.

Reply:
Fixed